macros/WebRTCSidebar.ejs

```diff
@@ -58,52 +58,66 @@ var text = mdn.localStringMap({
 <section id="Quick_Links">
  <ol>
   <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API"><strong><%=text['WebRTC_API']%></strong></a></li>
-  <li data-default-state="open"><a href="#"><strong><%=text['WebRTC_guides']%></strong></a>
-   <ol>
-     <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/Architecture"><%=text['Architecture']%></a></li>
-     <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/WebRTC_Basics"><%=text['Basics']%></a></li>
-     <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/Protocols"><%=text['Protocols']%></a></li>
-     <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/Connectivity"><%=text['Connectivity']%></a></li>
-     <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/Overview"><%=text['Overview']%></a></li>
-     <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/Session_lifetime"><%=text['Session_lifetime']%></a></li>
-     <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/Using_data_channels"><%=text['Using_data_channels']%></a></li>
-   </ol>
+  <li class="toggle">
```

Comment:
The ``{{WebRTCSidebar}}`` macro is correctly converted 👍, as seen on https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Signaling_and_video_calling ("WebRTC Basics" in the sidebar):

<img width="490" alt="pr 789 webrtcsidebar" src="https://user-images.githubusercontent.com/286017/45395909-fe252280-b5fd-11e8-88e2-cce12045a66c.png">